### PR TITLE
Search term should be URL-encoded params for search

### LINKED ISSUE #1985: Error while searching on T411
I was searching ChäoS;Child and I saw on my debug log this error because of the ";"

This should url-encoded, but it doesn't seems so..

img

Thanks a lot,

### PR DESCRIPTION
**Pull request by labrys**
Search term should be URL-encoded
Use params for search
Fix no results returned (thanks to @fernandog #1992)

- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Fixes: #1985

### CODE REVIEW

**Comment by fernandog** [comment]
@duramato did you test this?

File: medusa/providers/torrent/json/t411.py [hunk 1]
```diff
@@ -121,9 +138,12 @@ def parse(self, data, mode):
...
 
-        if not unsorted_torrent_rows or not isinstance(unsorted_torrent_rows, dict):
-            logger.log('Data returned from provider does not contain any {0}torrents'.format(
-                'confirmed ' if self.confirmed else ''), logger.DEBUG)
+        if not unsorted_torrent_rows:
+            logger.log(
+                'Data returned from provider does not contain any {torrents}'.format(
+                    torrents='confirmed torrents' if self.confirmed else 'torrents'
+                ), logger.DEBUG
+            )
```
Comment (fernandog) [inline medusa/providers/torrent/json/t411.py]: @labrys personally i like the old log "format".

**Comment by fernandog** [comment]
@medariox did you tested this? can you send logs please?

**Comment by medariox** [comment]
Yes. Logs for what?

**Comment by labrys** [comment]
@fernandog I tested it with the series Chaos;Child.  It works (as in no errors due to the semicolon and latin-1 characters) but returns no results since its not available on the indexer.  I also tested manual search and backlog with popular shows and got results.  I did not test daily search.

File: medusa/providers/torrent/json/t411.py (hunk 1 above)
Comment (labrys) [inline medusa/providers/torrent/json/t411.py]: Change it back if you prefer.  I personally think this is clearer, but it makes no real difference to me.

**Comment by fernandog** [comment]
This PR didn't fixed:
Issue is back: https://github.com/pymedusa/Medusa/issues/2117

**Comment by labrys** [comment]
@fernandog Thats an SSL issue.  Updating python to > 2.7.10 should fix it as well as running an updated version of OpenSSL.  There is likely other ways to fix it, but that shouldnt preclude the user from updating to avoid further SSL issues.

**Pull request merged**